### PR TITLE
Cleanup and consolidation

### PR DESCRIPTION
**Pull request by mmhh1910**
Created constants.py and refactored a few constants into it that have a global scope - especially 0x0.

### CODE REVIEW

**Review by JonathonDunford — DISMISSED** [review]
Thank you!

File: app/constants.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from web3 import Web3
+
+# Eternally constant constants
+
+ZERO_ADDR = "0x0000000000000000000000000000000000000000"
+ZERO_ADDR_BYTES = Web3.toBytes(hexstr=ZERO_ADDR)
```
Comment (freeatnet) [inline app/constants.py]: Can we import a smaller function for this operation?

Comment (mmhh1910) [reply app/constants.py]: Valid question.

**Review by JonathonDunford — APPROVED** [review]
Looks good. Verified bytes were correct also.

**Pull request merged**